automatic/datagrip/tools/ChocolateyInstall.ps1

```diff
@@ -0,0 +1,14 @@
+﻿$ErrorActionPreference  = 'Stop'
+
+$arguments              = @{
+    packageName         =  $env:ChocolateyPackageName
+    softwareName        = 'Datagrip'
```

Comment:
This is wrong.
This is meant to match the displayed name in Programs and Features.
In this case it should be changed to `JetBrains DataGrip*`.
Otherwise it will be broken for licensed users.